dev/com.ibm.ws.artifact.zip/src/com/ibm/ws/artifact/zip/cache/internal/ZipFileData.java

```diff
@@ -368,44 +372,44 @@ public void displayData() {
             openText =
-                "   Open: First [ " + toRelMilliSec(initialAt, firstOpenAt) + "ms ]" +
-                " Last [ " + toRelMilliSec(initialAt, lastOpenAt) + "ms ]" +
+                "   Open: First [ " + toRelSec(initialAt, firstOpenAt) + " (s) ]" +
+                " Last [ " + toRelSec(initialAt, lastOpenAt) + " (s) ]" +
                 " Count [ " + toCount(openCount) + " ]" +
-                " Duration [ " + toAbsMilliSec(openDuration) + "ms ]";
+                " Duration [ " + toAbsSec(openDuration) + " (s) ]";
         } else {
             openText =
-                "   Open: First [ " + toRelMilliSec(initialAt, firstOpenAt) + "ms ]" +
-                " Last [ " + toRelMilliSec(initialAt, lastOpenAt) + "ms ]" +
-                " Next Last [ " + toRelMilliSec(initialAt, lastLastOpenAt) + "ms ]" +
+                "   Open: First [ " + toRelSec(initialAt, firstOpenAt) + " (s) ]" +
+                " Last [ " + toRelSec(initialAt, lastOpenAt) + " (s) ]" +
+                " Next Last [ " + toRelSec(initialAt, lastLastOpenAt) + " (s) ]" +
                 " Count [ " + toCount(openCount) + " ]" +
-                " Duration [ " + toAbsMilliSec(openDuration) + "ms ]";
+                " Duration [ " + toAbsSec(openDuration) + " (s) ]";
         }
         System.out.println("ZFR " + openText);
 
         String pendingText =
-                "   Pending: First [ " + toRelMilliSec(initialAt, firstPendAt) + "ms ]" +
-                " Last [ " + toRelMilliSec(initialAt, lastPendAt) + "ms ]" +
+                "   Pending: First [ " + toRelSec(initialAt, firstPendAt) + " (s) ]" +
+                " Last [ " + toRelSec(initialAt, lastPendAt) + " (s) ]" +
                 " Count [ " + toCount(openToPendCount) + " ]";
         System.out.println("ZFR " + pendingText);
 
         String pendingBeforeOpenText =
                 "     Pending to Open: Count [ " + toCount(pendToOpenCount) + " ]" +
-                " Duration [ " + toAbsMilliSec(pendToOpenDuration) + "ms ]";
+                " Duration [ " + toAbsSec(pendToOpenDuration) + " (s) ]";
         System.out.println("ZFR " + pendingBeforeOpenText);
 
         String pendingBeforeCloseText =
                 "     Pending to Full Close: Count [ " + toCount(pendToFullCloseCount) + " ]" +
-                " Duration [ " + toAbsMilliSec(pendToFullCloseDuration) + "ms ]";
+                " Duration [ " + toAbsSec(pendToFullCloseDuration) + " (s) ]";
         System.out.println("ZFR " + pendingBeforeCloseText);
 
         String closeText =
-                "   Full Close: First [ " + toRelMilliSec(initialAt, firstFullCloseAt) + "ms ]" +
-                " Last [ " + toRelMilliSec(initialAt, lastFullCloseAt) + "ms ]";
+                "   Full Close: First [ " + toRelSec(initialAt, firstFullCloseAt) + " (s) ]" +
+                " Last [ " + toRelSec(initialAt, lastFullCloseAt) + " (s) ]";
         System.out.println("ZFR " + closeText);
```

Comment:
Looks like there are a few System.out.println's that were there before your changes - some where the output starts with "ZFR" - should these be in the product code?